config/routes.rb

```diff
@@ -0,0 +1,18 @@
+Rails.application.routes.draw do
+  devise_for :users
+  
+  resources :splash, only: [:index, :logout]
+  resources :group do
+    resources :payment
+  end
```

Comment:
- [ ] It will be great to ensure that you only add the necessary routes in your application. For example, when I check your groups controller I can see that you are not using the destroy route and yet it has been added. You can use the `expect` or the `only` methods to define the routes that you need. To learn more about this you can use this[ link ](https://guides.rubyonrails.org/routing.html#shallow-nesting)

Reply:
This change has not been implemented well